[BUG] After change to utils/rpm.mk (from #2901 and #2902), the server RPM build fails

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

@dbutenhof  in https://github.com/distributed-system-analysis/pbench/pull/2902#discussion_r907361461 points out that the new `targets.json` file that was added by PR #2901 (against main) and PR #2902 (against b0.71) causes a problem with the *server* RPM because  the `utils/rpm.mk` file (which is where the `targets.json` change was made) is used by by both agent and server. 

The simplest solution is to add an empty `targets.json` in the server/rpm directory.

### To Reproduce

On main, `cd .../server/rpm; make spec`.

### Actual Results

```
$ make spec
mkdir -p /home/nick/rpmbuild/BUILD /home/nick/rpmbuild/BUILDROOT /home/nick/rpmbuild/SPECS /home/nick/rpmbuild/SOURCES /home/nick/rpmbuild/SRPMS /home/nick/rpmbuild/RPMS
if [ -e ./seqno ] ;then expr 1 + 1 > ./seqno ;fi
jinja2 pbench-server.spec.j2 targets.json -D version=0.71.0 -D gdist=g4db6a5c80 -D seqno=1 > /home/nick/rpmbuild/SPECS/pbench-server.spec
Traceback (most recent call last):
  File "/usr/bin/jinja2", line 33, in <module>
    sys.exit(load_entry_point('jinja2-cli==0.7.0', 'console_scripts', 'jinja2')())
  File "/usr/lib/python3.9/site-packages/jinja2cli/cli.py", line 424, in main
    sys.exit(cli(opts, args))
  File "/usr/lib/python3.9/site-packages/jinja2cli/cli.py", line 262, in cli
    with open(path) as fp:
FileNotFoundError: [Errno 2] No such file or directory: '/home/nick/src/pbench/github/pbench/worktrees/correct-deps-agent-spec-backport/server/rpm/targets.json'
make: *** [../../utils/rpm.mk:44: spec] Error 1
```

### Expected Results

No error

### Additional information

_No response_

--- a/agent/rpm/pbench-agent.spec.j2
+++ b/agent/rpm/pbench-agent.spec.j2
@@ -19,10 +19,12 @@ Buildarch:      noarch
 Requires:  python39
 # This is used by the shebang processor
 %define __python3 /usr/bin/python3.9
+%define __python_name python39
 
 %else
 
 Requires:  python3
+%define __python_name python3
 
 %endif
 
@@ -34,7 +36,7 @@ Requires:  python3
 
 # RPMs for modules in requirements.txt
 Requires: python3-pip, python3-requests
-#RPMS for module dependencies
+# RPMS for module dependencies
 Requires: python3-psutil
 
 # RHEL7 also does not define __python3 - we need it for installing
@@ -43,18 +45,10 @@ Requires: python3-psutil
 
 %else
 
-{# Jinja2 template for python dependencies - see the targets.json file for the input data #}
-{%for tname, tcond in targets.items() -%}
-
-%if {{ tcond }}
 # RPMs for modules in requirements.txt
-Requires: {{ tname }}-pip, {{ tname }}-cffi, {{ tname }}-requests
-#RPMS for module dependencies
-Requires: {{ tname }}-psutil
-%endif
-
-{% endfor %}
-{# end of template #}
+Requires: %{__python_name}-pip, %{__python_name}-cffi, %{__python_name}-requests
+# RPMS for module dependencies
+Requires: %{__python_name}-psutil
 
 %endif
 

--- a/agent/rpm/targets.json
+++ b/agent/rpm/targets.json
@@ -1,7 +1,0 @@
-{
-    "targets": {
-        "python3": "0%{?rhel} != 8",
-        "python39": "0%{?rhel} == 8"
-    }
-}
-

--- a/utils/rpm.mk
+++ b/utils/rpm.mk
@@ -41,7 +41,7 @@ srpm: spec patches tarball
 .PHONY: spec
 spec: rpm-dirs ${prog}.spec.j2
 	if [ -e ./seqno ] ;then expr ${seqno} + 1 > ./seqno ;fi
-	jinja2 ${prog}.spec.j2 targets.json -D version=${VERSION} -D gdist=g${sha1} -D seqno=${seqno} > ${RPMSPEC}/${prog}.spec
+	jinja2 ${prog}.spec.j2 -D version=${VERSION} -D gdist=g${sha1} -D seqno=${seqno} > ${RPMSPEC}/${prog}.spec
 	cp ${PBENCHTOP}/utils/rpmlint ${RPMSPEC}/pbench-common.rpmlintrc
 	XDG_CONFIG_HOME=${PBENCHTOP}/utils rpmlint ${RPMSPEC}/${prog}.spec
 

